cpp/src/arrow/flight/transport/grpc/grpc_client.cc

```diff
@@ -316,7 +311,6 @@ class FinishableDataStream : public internal::ClientDataStream {
 
   std::shared_ptr<ClientRpc> rpc_;
   std::shared_ptr<Stream> stream_;
-  std::shared_ptr<MemoryManager> memory_manager_;
```

Comment:
Is it possible to also remove `memory_manager_` from transport server implementations?

Reply:
It's already removed from the transport server implementations (the only reference there is to pass it on to the base class which does the heavy lifting)